blueprints/app/files/.travis.yml

```diff
@@ -9,6 +9,17 @@ cache:
   directories:
     - node_modules
 
+env:
+  - E_SCENARIO=ember-1.10
```

Comment:
I submitted https://github.com/kategengler/ember-try/pull/21 to make the default config more in line with this.

Could you add the `"default"` scenario in the `config/ember-try.js` (like from my PR above) instead of `ember-1.10`?